Exercises/01/Begin/CMakeLists.txt

```diff
@@ -1,7 +1,15 @@
 cmake_minimum_required(VERSION 3.16)
 project(KokkosTutorial01)
-include(../../common.cmake)
+
+# Add a custom module path for find_package
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/../../.cmake)
```

Comment:
```suggestion
list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../../.cmake")
```

Reply:
That's better indeed, I will update all the files.